### ISSUE TITLE
Update to latest version of Solidity

### ISSUE THREAD

**Issue by samajammin** [SuperblocksHQ/ethereum-studio#244]
## Summary

Currently the compiler is hardcoded to `0.5.10` & all templates use that version.

## Motivation

Since Ethereum Studio's core purpose is to onboard Ethereum developers, we should be using the latest & best (stable) tools available. We shouldn't teach approaches & patterns that will change when they go to set up a local dev environment.

## Describe alternatives you've considered
None

## Additional context
None

**Comment by javier-tarazaga** [comment]
We always end up running into this and would be much better to concentrate on #245 so we have a good way to actually be smart about it and use the right solidity version.

**Comment by ChrisChinchilla** [comment]
I was sure I already updated to 0.6 somewhere? Or am I being very forgetful?

**Comment by samajammin** [comment]
> I was sure I already updated to 0.6 somewhere? Or am I being very forgetful?

@ChrisChinchilla nope, as I mentioned, the project is currently hardcoded to use `0.5.10`.

**Comment by ChrisChinchilla** [comment]
I don't deny that, I just feel that I submitted an update to this already 🤔

**Comment by ChrisChinchilla** [comment]
Maybe it was to 0.5.x
